default/stringtables/en.txt

```diff
@@ -1939,6 +1945,18 @@ Join a game
 REFRESH_LIST_BN
 Refresh list
 
+##
+## Password Dialog
+##
+
+AUTHENTICATION_WINDOW_TITLE
+Authentication
+
+AUTHENTICATION_DESC
+This player name requires authentication. Reminder! Password will be transfered plain-text.
```

Comment:
You should really crypto-hash the password before sending...

Reply:
Which crypto library could be add to dependencies?

Reply:
Also http://freeorion.org/forum/viewtopic.php?f=9&t=10661

Reply:
@geoffthemedio, wouldn't encryption be a bit of an overkill here? As @adrianbroher mentioned in the forum thread @o01eg linked to, this is just a game, not some important private data that needs proper protection.

Reply:
The concern is that someone will reuse a password for something that is important. But as long as it is clearly indicated that it is sent unencrypted / plaintext, it's OK I suppose...

Reply:
The concern is that someone will reuse a password for something that is important. But as long as it is clearly indicated that it is sent unencrypted / plaintext, it's OK I suppose...

Reply:
I think it possible to have some OTP or even web-authentication but it will be later work.